lib/STPManager/STPManager.cpp

```diff
@@ -208,12 +208,12 @@ ASTNode STPMgr::CreateBVConst(unsigned int width,
   // number of bits in unsigned long. The variable "copied" keeps
   // track of the number of chunks copied so far
 
-  const int shift_amount = sizeof(unsigned long) * 8;
+  constexpr int shift_amount = sizeof(unsigned long) * 8;
   while (copied + shift_amount < width)
   {
     CONSTANTBV::BitVector_Chunk_Store(CreateBVConstVal, shift_amount, copied,
                                       c_val);
-    if (shift_amount < (sizeof(bvconst) * 8))
+    if constexpr (shift_amount < (sizeof(bvconst) * 8))
```

Comment:
Now checked at compile time to avoid warning that `bvconst` size is less then `shift_amount` number.